ci/e2e-tests.yml

```diff
@@ -0,0 +1,113 @@
+name: 'dhis2: e2e tests'
+
+# Requirements:
+#
+# - Secrets:
+#     GITHUB_TOKEN
+#     CYPRESS_RECORD_KEY
+#     CYPRESS_DHIS2_USERNAME
+#     CYPRESS_DHIS2_PASSWORD
+#
+# - Customize environment variables:
+#     BASE_URL_INSTANCES: Set the base url for the instances, e.g. https://test.e2e.dhis2.org
+#     NAME_PATTERN_LEGACY_INSTANCES: Set the name pattern for your instances. {version} will be replaced by "[majorVersion].[minorVersion]"
+#     NAME_PATTERN_DEV_INSTANCE: Set the name pattern for your dev instance. {version} will be replaced by "dev"
+#
+# - Other optional customizations:
+#     containers: Set the matrix containers array for the cypress job. The number of parallel Cypress instances running for each backend version will equal the array length.
+#
+# - Set status check as required:
+#     We recommend setting "e2e-tests-success"  as a required step in your workflow.
+
+on:
+    pull_request:
+        types: [opened, synchronize, reopened, ready_for_review]
+
+env:
+    BASE_URL_INSTANCES: https://test.e2e.dhis2.org
+    NAME_PATTERN_LEGACY_INSTANCES: '{version}dev'
+    NAME_PATTERN_DEV_INSTANCE: dev
+
+concurrency:
+    group: ${{ github.workflow}}-${{ github.ref }}
+    cancel-in-progress: true
+
+defaults:
+    run:
+        shell: bash
+
+jobs:
+    compute-versions:
+        if: github.event.pull_request.draft == false
```

Comment:
Should the e2e tests run on forks? The e2e job in the verify-app workflow doesn't allow it: 
```if: '!github.event.push.repository.fork'```

Reply:
Makes sense. Disabled for forks.